content/terms/premium-support.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: markdown
+title: Terms of Service for Sourcegraph Premium Support
+---
+
+Last Modified: May 5, 2023
+
+This document outlines the basic terms and conditions that apply to Sourcegraph's Premium Support Offerings.
+
+## Premium Support
+
+### Benefits of Premium Support
+Purchasing Premium Support provides the following services beyond those provided by Sourcegraph's standard support model:
+- **Accelerated Service Level Agreements (SLAs)** - Accelerated triaging, faster initial response times, and queue priority lead reduced times to resolution.
+- **Elevated Support Expertise** - Named, dedicated primary and backup senior support engineers possessing deep knowledge and understanding of you specific organizations setup and tech stack with the ability to assist in the resolution of issues that arise.
```

Comment:
We are no longer referring it as primary and backup but simply two senior support engineers.